charts/covidDataExplorer/CovidChartUrl.ts

```diff
@@ -322,16 +372,38 @@ export class CovidConstrainedQueryParams extends CovidQueryParams {
         if (!hasMetric) this.casesMetric = true
     }
 
-    @computed get allowEverything() {
+    get allowEverything() {
         return false
     }
 
-    @computed get available() {
+    get rollingMultiplier() {
+        if (this.isWeekly) return 7
```

Comment:
Would maybe be nice to isolate a "DataExplorerOptionsState" class from the URL class and have all these derivations in there and keep the URL class simple with just the state of the query params.